client/tailwind.config.js

```diff
@@ -55,6 +55,8 @@ module.exports = {
       /* allow system colours https://www.w3.org/TR/css-color-4/#css-system-colors */
       LinkText: 'LinkText',
       ButtonText: 'ButtonText',
+      white: '#FFF',
+      black: '#000',
```

Comment:
We already have white and black colors defined so I assume those two are useless?